src/frontend/src/catalog/sink_catalog.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risingwave_pb::catalog::Sink as ProstSink;
+
+use super::SinkId;
+
+/// this struct `SinkCatalog` is used in frontend and compared with `ProstSink` it only maintains
+/// information which will be used during optimization.
+#[derive(Clone, Debug)]
+pub struct SinkCatalog {
```

Comment:
We will not support querying on sinks in the short term, at least not in the mvp. So pls remove this file for now.